tests/storage/hotplug.go

```diff
@@ -948,6 +949,44 @@ var _ = SIGDescribe("Hotplug", func() {
 				Entry("[Serial] with VMs and block", Serial, addDVVolumeVM, removeVolumeVM, corev1.PersistentVolumeBlock, false),
 			)
 
+			It("should allow to hotplug 75 volumes simultaneously", func() {
```

Comment:
Hi @awels - just wondering what was the reasoning for going with 75 volumes here? 

This is one of the longest running e2e tests that we have with it taking on average over 8 minutes to run. 

Would reducing the number of volumes to 15 or 20 harm the value of this test?

Reply:
So this test is to ensure adding/removing volumes still works properly when you get to a large number of already attached volumes. Once you get over 50 attached volumes, creating a pod exponentially becomes more expensive (not sure why). The original ask was for 250 volumes, but that takes a super long time and reducing it to about 75 was a happy medium, where it would exercise the code and not take a ridiculous long time. From what I remember the longest part of this test is actually creating the volumes, not the attaching the volumes to the VM. I believe HPP would be significantly faster when creating the volumes, however we don't have it available in the CI lane here. And we don't have enough local volumes to get to 75.